### DISCUSSION TITLE
Version for batocera?

### DISCUSSION THREAD

**Discussion by adamsbuilds** [webmin/authentic-theme#1630] [General]
Sorry wrong github

**Comment by swelljoe** [comment]
That's not a theme issue. That's a Webmin issue.

What distro is Batocera based on? If it's sufficiently similar to some supported distro, you may be able to just tell Webmin your OS is that distro and it'll work. (And, if it does, we can add it to the OS list.)

If Batocera is not a derivative of some existing distro, we'd need someone to volunteer to create the configs. Usually no (or very little) new code is needed to add support for a new distro, it's just config files to tell Webmin where everything lives and how to interact with the system (e.g. systemd or initscripts or something else, config file locations, etc.)

**Comment by swelljoe** [comment]
What I mean by the configs is like this: https://github.com/webmin/webmin/blob/master/apache/config-Rocky-Linux-7.0-ALL

Every module has a bunch of those files telling Webmin where to find things and how to start/stop/etc. the services it manages. To add support for an OS is to create those configs (plus adding a way to detect the OS in os_list.txt).

I won't add support for Batocera, but you can, and we'll be happy to review and merge your PR to do so.

**Comment by swelljoe** [comment]
I think maybe we're talking past each other (and I think maybe you're confused about what Authentic Theme is). Authentic is a theme for Webmin. It is not a Gnome or KDE theme (or any other desktop environment). It runs in a web browser when you visit a Webmin system...the theme has nothing to do with OS support; all operating systems supported by Webmin can use Authentic Theme.

**Comment by iliaross** [comment]
If I have time I will check on Batocera.

@adamsbuilds Do you know which Batocera linux is based on (i.e. Ubuntu, RHEL, BSD or ..?). Also, what it's purpose? To let people play old times console games?

**Comment by swelljoe** [comment]
No. @iliajie. OP isn't even talking about Webmin. It's some other Authentic theme.
